include/cuco/detail/static_map/static_map.inl

```diff
@@ -318,11 +319,47 @@ void static_map<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Stora
   auto const num = cuco::detail::distance(first, last);
   if (num == 0) { return; }
 
-  auto const grid_size = cuco::detail::grid_size(num, cg_size);
-
-  static_map_ns::detail::insert_or_apply<cg_size, cuco::detail::default_block_size()>
-    <<<grid_size, cuco::detail::default_block_size(), 0, stream.get()>>>(
-      first, num, op, ref(op::insert_or_apply));
+  int32_t constexpr shmem_block_size = 1024;
+  int32_t const default_grid_size    = cuco::detail::grid_size(num, cg_size);
+
+  int32_t constexpr cardinality_threshold   = shmem_block_size;
+  int32_t constexpr shared_map_num_elements = cardinality_threshold + shmem_block_size;
+  float constexpr load_factor               = 0.7;
```

Comment:
question: 0.7 occupancy seems a bit too high for the default linear probing. I know it's used by Miloni's work, but have you evaluated the perf diff between 0.5 and 0.7?

Reply:
I haven't tested that out. I will try that and post the results here.

Reply:
When slot type is `cuco::pair<int64_t, int64_t>` and with **LF = 0.5**, the shared_map will consume more static shared memory around 65KiB, which is unavailable for most GPUs. With LF as 0.7, max storage used is around 45.7 KiB, which is sufficient most GPUs.

Reply:
Thanks for testing. Let's stick with the current setup then.